index.html

```diff
@@ -1,184 +1,288 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Microverse Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <nav>
-            <a href="#">My Logo</a>
-            <img src="./assets/menu.png" alt="logo">
-        </nav>
-        <div class="background">
-            <div class="intro">
-                <div class="text">
-                    <h1>I'm Favour. <br> Glad to see you!</h1>
-                </div>
-                <p class="intro-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                
-                <div class="socials">
-                    <h5>LET'S CONNECT</h5>
-                    <ul class="social-icon">
-                        <li> <a href="#"> <img src="./assets/twitter.svg" alt="twitter"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/medium.svg" alt="medium"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/github.svg" alt="github"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/angellist.svg" alt="angellist"> </a> </li>
-                    </ul>
-                </div>
-            </div>
+      <nav>
+        <a href="#">My Logo</a>
+        <div class="link">
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+          <img src="./assets/menu.png" alt="logo" />
+        </div>
+      </nav>
```

Comment:
- [ ] As suggested by the previous please make sure you're navbar is looking like the given design, there should be some spacing to the left and right. please revisit the previous comment below it was explained clearly:
> * [ ]  Kindly consider changing this to `min-height: 80vh;` as 140 makes your design different from the original design.
>       your design
>       ![qw](https://user-images.githubusercontent.com/96683843/236251702-8081f7fe-6edd-4c13-ac48-8778ac217434.jpg)
>       figma design
>       ![v](https://user-images.githubusercontent.com/96683843/236251796-6a83dc94-2467-4e02-9a57-86acce218c4e.jpg)